plugin/src/main/java/io/jenkins/plugins/analysis/warnings/DifferentCompatibilityScanner.java

```diff
@@ -0,0 +1,170 @@
+package io.jenkins.plugins.analysis.warnings;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import edu.hm.hafner.analysis.Issue;
+import edu.hm.hafner.analysis.Report;
+import edu.hm.hafner.analysis.RevApiInfoExtension;
+import hudson.model.Run;
+
+import io.jenkins.plugins.analysis.core.model.AnalysisModelParser;
+import io.jenkins.plugins.analysis.core.model.DescriptionProvider;
+import io.jenkins.plugins.analysis.core.model.DetailsTableModel;
+import io.jenkins.plugins.analysis.core.model.FileNameRenderer;
+import io.jenkins.plugins.analysis.core.model.StaticAnalysisLabelProvider;
+import io.jenkins.plugins.analysis.core.model.StaticAnalysisLabelProvider.AgeBuilder;
+import io.jenkins.plugins.datatables.TableColumn;
+import io.jenkins.plugins.util.JenkinsFacade;
+
+public class DifferentCompatibilityScanner extends AnalysisModelParser {
```

Comment:
Why is this in a separate class and not part of `RevApi`. There is no other implementation planned.